src/shogun/mathematics/linalg/LinalgBackendViennaclKernels.h

```diff
@@ -90,6 +90,47 @@ namespace shogun
 		return kernel;
 	}
 
+	/** Generates the set_rows_const computation kernel
+	 * The OpenCL kernel that helps to set rows const to SGMatrix from SGVector
+	 */
+	template <class T>
+	static viennacl::ocl::kernel& generate_set_rows_const_kernel()
+	{
+		std::string kernel_name = "set_rows_const_" +
+			linalg::implementation::ocl::get_type_string<T>();
+
```

Comment:
This needs to be tested. Is it tested on your machine? Travis cannot check it...